python_modules/dagster/dagster_tests/components_tests/code_locations/python_script_location/scripts/defs.yml

```diff
@@ -0,0 +1,11 @@
+component_type: collection
+
+component_params:
+  component_type: python_script
+  components:
+    script_one:
+      - key: a
+      - key: b
+        deps: [up1, up2]
+    script_three:
+      - key: override_key
```

Comment:
Now that I see this I think it is important to restructure this schema to have an `asset_specs` key.

```yaml
component_type: collection
component_params:
  component_type: python_script
  components:
    script_one:
       assets:
          - key: a
          - key: b
             deps: [up1, up2]
    script_three:
       assets:
           - key: override_key
```